docs/HyperIndex/Guides/rpc-requests.md

```diff
@@ -0,0 +1,324 @@
+---
+id: rpc-requests
+title: RPC Requests
+sidebar_label: RPC Requests
+slug: /rpc-requests
+---
+
+# RPC Requests guide
+
+TLDR; The repo for the code base can be found [here](todo_add_link)
```

Comment:
Let's set it to this, and when we transfer the repo we can make sure it matches the below :)
```suggestion
TLDR; The repo for the code base can be found [here](https://github.com/enviodev/rpc-token-data-example)
```